kolibri/auth/models.py

```diff
@@ -78,6 +80,18 @@ def get_full_name(self):
     def get_short_name(self):
         return self.first_name
 
+    def has_perm(self, perm, obj=None):
+        return _user_has_perm(self, perm, obj)
```

Comment:
This may be indefensible, I'll let you decide, but the function _user_has_perm checks permissions using each installed backend -- which is indeed exactly the behavior I want. Seems unlikely to change in Django, too. More or less the same thing is done by Django's BaseUser, however I chose not to subclass that model because it defines fields we have no intention of using, and there's no good way I know of to preclude inherited fields.